### PR TITLE
Get real ip from proxy headers

### PR DESCRIPTION
**Pull request by felixbarny**

### CODE REVIEW

File: apm-agent-plugins/apm-servlet-plugin/src/main/java/co/elastic/apm/servlet/ClientIpUtils.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+public class ClientIpUtils {
+
+    /**
+     * This method returns the first IP from common HTTP header names used by reverse proxies.
+     * If there is no such header, it returns {@link HttpServletRequest#getRemoteAddr()}
+     *
+     * @param request The HTTP request.
+     * @return the
+     */
+    public static String getRealIp(HttpServletRequest request) {
```
Comment (jalvz) [inline apm-agent-plugins/apm-servlet-plugin/src/main/java/co/elastic/apm/servlet/ClientIpUtils.java]: for reference, this is the apm-server implementation: https://github.com/elastic/apm-server/blob/12d38ced19a8a4a9555435e960bc47da4f824c6f/utility/ip.go#L12

Comment (felixbarny) [reply apm-agent-plugins/apm-servlet-plugin/src/main/java/co/elastic/apm/servlet/ClientIpUtils.java]: As the apm server is doing this anyway, I'll close this PR.

**Comment by felixbarny** [comment]
Turns out, the APM server only detects the real IP of the agent, not the IP of the actual user

**Comment by alvarolobato** [comment]
Are you finally closing this? needs review?

**Pull request merged**